cranelift/filetests/filetests/verifier/ref-safepoint.clif

```diff
@@ -0,0 +1,12 @@
+test verifier
+target x86_64
+
+function %test(r64) -> r64 { 
+    block0(v0: r64): 
+        v1 = null.r64 
+        v2 = is_null v0 
```

Comment:
There should be an error annotation in this test, so it'd check that the verifier asserts because a ref type is used yet ref types aren't enabled, right?